File: python/tasks/30-days/06-review-loop.py
# Enter your code here. Read input from STDIN. Print output to STDOUT
T = int(raw_input())
for i in range(T):
    output = ['', '']
    S = raw_input()
    for c in range(len(S)):
        if c % 2 == 0:
            output[0] += S[c]
        else:
            output[1] += S[c]
    print ' '.join(output)

File: js/tasks/30-days/06-review-loop.js
function processData(input) {
    //Enter your code here
    let inputString = input.split("\n");
    let T = inputString[0];
    inputString.slice(1).forEach((S) => {
        let output = ['', ''];
        for (let c = 0; c < S.length; c++) {
            if (c % 2 === 0) {
                output[0] += S[c]; 
            } else {
                output[1] += S[c]; 
            }
        }
        console.log(output.join(' '));
    });
} 

process.stdin.resume();
process.stdin.setEncoding("ascii");
_input = "";
process.stdin.on("data", function (input) {
    _input += input;
});

process.stdin.on("end", function () {
   processData(_input);
});

